computation_graph/graph.py

```diff
@@ -148,6 +148,29 @@ def replace_source(
     )
 
 
+@gamla.curry
+def replace_source(
+    original: base_types.CallableOrNode,
+    replacement: base_types.CallableOrNodeOrGraph,
+    current_graph: base_types.GraphType,
+) -> base_types.GraphType:
+    if make_computation_node(original) not in get_all_nodes(current_graph):
```

Comment:
do we also want to raise some type of error? maybe not breaking but just so we know we are trying to replace a node that doesn't exist?

Reply:
no, because we want it to not have an effect, sometimes you don't know if it's in the graph, which is fine I think.